components/css/styles.css

```diff
@@ -3,20 +3,22 @@
 @import url("../text/css/styles.css");
 @import url("../lists/css/styles.css");
 @import url("../rating/css/styles.css");
+@import url("../../css/styles.css");
 
 :root {
   --primary-color: #ff3b30;
   --secondary-color: #fff6ea;
   --action-color: #3b82f6;
   --gray-color: #9a9a9a;
   --black-color: #282828;
-  --white-color: #efefef;
+  --white-color: #ffffff;
   --lynx-white-color: #f7f7f7;
-  --violet-color: #534837;
+  --violet-color: #5348c7;
   --lynx-gray: #c5c5c5;
 
   --green-color: #596;
   --yellow-color: #ffff00;
+  --footer-color: #333;
```

Comment:
You should name these variables in a more generic way, like which color is getting used for which thing than it would be easy for you to change them later on.

Reply:
Moving forward I will make the changes as you suggested when I use one colour for single purpose.